includes/WpGears/Gearman/Worker.php

```diff
@@ -77,6 +77,8 @@ public function work() {
 			$result = false;
 		}
 
+		do_action( 'wp_async_after_worker_works', $result, $this );
```

Comment:
All of the other actions start with `wp_async_task_` so for consistency, we should prefix this with that as well. Maybe just `wp_async_task_after_work`